cmd/query_test.go

```diff
@@ -16,36 +14,31 @@ import (
 
 var _ = Describe("Query", func() {
 	var (
-		server   *ghttp.Server
-		token    string
-		tempHome string
-		stdin    bytes.Buffer
-		command  *exec.Cmd
+		server       *ghttp.Server
+		token        string
+		tempSettings *temporarySettings
+		stdin        bytes.Buffer
+		command      *exec.Cmd
 	)
 
 	BeforeEach(func() {
 		server = ghttp.NewServer()
 
-		var err error
-		tempHome, err = ioutil.TempDir("", "circleci-cli-test-")
-		Expect(err).ToNot(HaveOccurred())
+		tempSettings = withTempSettings()
 
 		token = "mytoken"
-		command = exec.Command(pathCLI, "query", "-",
+		command = commandWithHome(pathCLI, tempSettings.home,
```

Comment:
I missed the PR when this was introduced, but still nice helper function. 👏

Reply:
This was refactored here:
https://github.com/CircleCI-Public/circleci-cli/pull/248/commits/8937c9b09af515e5501cfdcc4c22381c2ec670bb